pkgs/tools/wayland/hyprevents/default.nix

```
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
```

Nit: Separate lines makes this easier to diff later
```suggestion
{ fetchgit
, lib
, makeWrapper
, stdenv
, hyprland
, bash
}:
```